src/style.css

```diff
@@ -47,5 +48,12 @@ body {
 }
 
 .swiper{
-    height: 500px;
+    height: 600px;
 }
+
+.swiper-pagination-bullet{
+    box-shadow: 0 0 30px black;
+}
+
+
+
```

Comment:
get rid off unnecessary breaks at the end of the file (always leave only 1 empty line)